website/docs/r/ec2_transit_gateway_multicast_domain.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "EC2"
+layout: "aws"
+page_title: "AWS: aws_ec2_transit_gateway_multicast_domain"
+description: |-
+  Manages an EC2 Transit Gateway Multicast Domain
+---
+
+# Resource: aws_ec2_transit_gateway_multicast_domain
+
+Manages an EC2 Transit Gateway Multicast Domain.
+
+## Example Usage
```

Comment:
Can we include another set of larger examples that show the TGW, TGW-Multicast, Ec2 instance all working together?

https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/contribution-checklists.md#documentation-update

Reply:
Added a full example